### PR TITLE
Wargaming.GameCenter version 21.7.3.7062

### PR DESCRIPTION
**Pull request by solomoncyj**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34893)

### CODE REVIEW

File: manifests/w/Wargaming/GameCenter/21.07.02.7005/Wargaming.GameCenter.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.07.02.7005
```
Comment (ItzLevvie) [inline manifests/w/Wargaming/GameCenter/21.07.02.7005/Wargaming.GameCenter.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 21.7.3.7062
```

File: manifests/w/Wargaming/GameCenter/21.07.02.7005/Wargaming.GameCenter.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.07.02.7005
```
Comment (ItzLevvie) [inline manifests/w/Wargaming/GameCenter/21.07.02.7005/Wargaming.GameCenter.installer.yaml:5]:
```suggestion
PackageVersion: 21.7.3.7062
```

File: manifests/w/Wargaming/GameCenter/21.07.02.7005/Wargaming.GameCenter.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Wargaming.GameCenter
+PackageVersion: 21.07.02.7005
```
Comment (ItzLevvie) [inline manifests/w/Wargaming/GameCenter/21.07.02.7005/Wargaming.GameCenter.yaml:5]:
```suggestion
PackageVersion: 21.7.3.7062
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You will have to update the version folder from `21.07.02.7005` to `21.7.3.7062` using Visual Studio Code by pressing `.` (dot) on your keyboard in this PR.

**Comment by solomoncyj** [comment]
done, but the updates does not want to push

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**